test/PlaidML/OpTest.Sum.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-cpu.pp \
+// RUN:                                       --runner mlir-cpu-runner -e main \
+// RUN:                                       --shared-libs=%mlir_runner_utils \
+// RUN:                                       --entry-point-result=void | FileCheck %s
+// RUN-GPU: %python_executable %imex_runner -i %s --pass-pipeline-file=%p/linalg-to-llvm.pp \
```

Comment:
I would prefer just having the RUN command here and disabling them from lit.config